### PR TITLE
[CI environment] Cleanup outdated custom removal logic for VWan

### LINKED ISSUE #1492: [Feature Request]: Cleanup outdated custom removal logic for VWan
### Description

After VWan module restructuring, the related custom removal logic
```
'virtualWans' {
                    $removalSequence += @(
                        'Microsoft.Network/vpnGateways',
                        'Microsoft.Network/virtualHubs',
                        'Microsoft.Network/vpnSites'
                    )
                    break
                }
```
is not needed anymore.

Clean up the code and update Wiki accordingly.

### PR DESCRIPTION
**Pull request by eriqua**
# Description

Close #1492 
After VWan module restructuring, the custom removal logic is not needed anymore. 
Cleaning up the code and updating Wiki accordingly.

| Pipeline |
| - |
| [Network: VirtualWans](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.virtualwans.yml) |

# Type of Change

Please delete options that are not relevant.

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

# Checklist

- [ ] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (readme)
- [ ] I did format my code

### CODE REVIEW

File: utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1 [hunk 1]
```diff
@@ -73,17 +73,19 @@ function Initialize-DeploymentRemoval {
...
-                    )
-                    break
-                }
-                ### CODE LOCATION: Add custom removal sequence here
-            }
+
+            ### CODE LOCATION: Add custom removal sequence here
+            ## Add custom module-specific removal sequence following the example below
+            # switch ($moduleName) {
+            #     '<moduleName01>' {
```
Comment (MrMCake) [inline utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1:80]:
```suggestion
            #     '<moduleName01>' {  // For example: 'virtualWans'
```

File: utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1 [hunk 2]
```diff
@@ -73,17 +73,19 @@ function Initialize-DeploymentRemoval {
...
+            #         $removalSequence += @(
+            #             '<resourceType01>',
```
Comment (MrMCake) [inline utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1:82]:
```suggestion
            #             '<resourceType01>', // For example 'Microsoft.Network/vpnSites'
```

File: utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1 (hunk 1 above)
Comment (eriqua) [reply utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1]: check

File: utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1 (hunk 2 above)
Comment (eriqua) [reply utilities/pipelines/resourceRemoval/Initialize-DeploymentRemoval.ps1]: check

**Review by MrMCake — APPROVED** [review]

**Pull request merged**